src/wallet/components/WalletBasic.tsx

```diff
@@ -0,0 +1,26 @@
+import { Children, useMemo } from 'react';
+import { findComponent } from '../../core-react/internal/utils/findComponent';
+import { cn } from '../../styles/theme';
+import type { WalletReact } from '../types';
+import { ConnectWallet } from './ConnectWallet';
+import { WalletDropdown } from './WalletDropdown';
+import { useWalletContext } from './WalletProvider';
+
+export function WalletBasic({ children, className }: WalletReact) {
```

Comment:
What's the difference between `WalletBasic` and `WalletDefault`?

Reply:
WalletBasic is just a wrapper that destructures children and reorganizes them. 

WalletDefault implements WalletBasic, and looks like this:
<img width="342" alt="image" src="https://github.com/user-attachments/assets/3c62fd3b-3fe2-4a6b-a2d9-1bee2202beec" />

But WalletBasic can be extended beyond WalletDefault, like this:
<img width="347" alt="image" src="https://github.com/user-attachments/assets/eecef473-910c-4e79-8101-1ba91f933575" />

I'm not wedded to these names, perhaps WalletDefault should become WalletBasicDefault or something.